napari/plugins/utils.py

```diff
@@ -0,0 +1,67 @@
+from . import PluginError, log_plugin_error
+from . import plugin_manager as napari_plugin_manager
+from ._hookexec import _hookexec
+from typing import Optional
+from ..types import LayerData
+
+
+def get_layer_data_from_plugins(
```

Comment:
This kind of feels less generic than `utils.py`. Should this perhaps live in `plugins/io_plugins.py`? I might also rename it `read_data_with_plugins`.

Reply:
I love the function though, quite easy to follow. =)

Reply:
yeah, I have also felt like it didn't quite belong there.  I'd be happy to create a new module for it and rename it.